NektarDriftwave/Project.toml

```diff
@@ -0,0 +1,27 @@
+name = "NektarDriftwave"
+uuid = "779ecde5-d1cc-4058-a649-58afcbbc11ce"
+authors = ["Matt Graham"]
+version = "0.1.0"
+
+[deps]
+CSV = "336ed68f-0bac-5ca0-87d4-7b16caf5d00b"
+Distributions = "31c24e10-a181-5473-b8eb-7969acd0382f"
+Gmsh = "705231aa-382f-11e9-3f0c-b7cb4346fdeb"
+HDF5 = "f67ccb44-e63f-5c2f-98bd-6dc0ccc4ba2f"
+MPI = "da04e1cc-30fd-572f-bb4f-1f8673147195"
+PDMats = "90014a1f-27ba-587c-ab20-58faa44d9150"
+ParticleDA = "61cd1fb4-f4c4-4bc8-80c6-ea5639a6ca2e"
+Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
+Tables = "bd369af6-aec1-5ad0-b16a-f7cc5008161c"
+XML = "72c71f33-b9b6-44de-8c94-c961784809e2"
+
+[compat]
+CSV = "0.10"
+Distributions = "0.25"
+Gmsh = "0.1"
+HDF5 = "0.16, 0.17"
+MPI = "0.20"
+PDMats = "0.11"
+ParticleDA = "1.1"
+Tables = "1.11"
+XML = "0.3"
```

Comment:
@giordano does this looks reasonable in terms of specifying `compat` constraints? I've pinned to minor versions of packages which I was using in locally and have found to work, other than HDF5 which based on previous discussion I think v0.17 should also be safe (I was using v0.16 locally).

Reply:
Yes, that's pretty much what I do as well.